app/views/profiles/show.html.erb

```diff
@@ -74,7 +74,24 @@
     </div>
     <% if @profiles_available %>
       <div class="tab-pane" id="profile-firecloud" role="tabpanel">
-        <%= render partial: 'user_firecloud_profile' %>
+        <% if current_user.registered_for_firecloud %>
+          <%= render partial: 'user_firecloud_profile' %>
+        <% else %>
+          <div class="container">
+            <div class="bs-callout bs-callout-danger">
+              <h4>Please complete your Terra registration</h4>
+              <p>
+                You may not update your Terra profile until you have registered with Terra and accepted the terms of service.
+                Please visit <%= link_to 'Terra', 'https://app.terra.bio', target: :_blank %>, select 'Sign in with Google'
+                from the top-lefthand nav menu, and complete the sign in and registration process.
```

Comment:
See other comment beginning "The help article on this would be more tailored".

Reply:
f3422a0